bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java

```diff
@@ -0,0 +1,25 @@
+package com.ayeeti.blservice.user;
+
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/users")
+public class UserController {
```

Comment:
You are missing DELETE and PUT endpoints here.

Reply:
Done.